src/ConfigWidgets/SymbolLocationsDialog.cpp

```diff
@@ -411,7 +412,8 @@ void SymbolLocationsDialog::SetUpModuleHeadlineLabel() {
   ORBIT_CHECK(module_.has_value());
   ui_->moduleHeadlineLabel->setVisible(true);
   ui_->moduleHeadlineLabel->setText(
-      QString(kModuleHeadlineLabel).arg(QString::fromStdString(module_.value()->name())));
+      QString(kModuleHeadlineLabel)
+          .arg(QString::fromStdString(std::filesystem::path(module_.value()->file_path()).filename())));
```

Comment:
`filename()` returns a std::filesystem::path. Is it working here? Maybe `.string()` is needed.

Reply:
As far as I can see, there is an implicit conversion happening. Do you want me to make this explicit?

Reply:
It's fine to keep it as it is now. Just a bit curious.

BTW, it seems that a lot of data view tests need to be updated 🙈

Reply:
Okay, I'll keep it then, as this is also how `filename` was used before.

I think I fixed the tests now :)

Reply:
Just for context and to clarify this. As you already figured out, windows needs the .string() otherwise it won't compile. This is because the underlying default type on windows is not std::string, but a utf16 string.